ui/src/app/views/concept-add-modal.tsx

```diff
@@ -125,9 +125,10 @@ export const ConceptAddModal = withCurrentWorkspace()
         addedIds: conceptIds
       };
       try {
-        await conceptSetsApi().createConceptSet(namespace, id, request);
+        const createdConceptSet =
+          await conceptSetsApi().createConceptSet(namespace, id, request);
```

Comment:
Just curious, is there a difference/benefit to using async/await here instead of putting the next two lines in `.then()`?

Reply:
Not particularly, I do think its more readable though, if there's no need to do other work in a synchronous manner.